addons/stock/i18n/stock.pot

```diff
@@ -6618,6 +6618,14 @@ msgid ""
 "instead."
 msgstr ""
 
+#. module: stock
+#: code:addons/stock/models/product.py:0
+#, python-format
+msgid ""
+"This product's company can not be changed as long as there is quantity for it "
```

Comment:
```suggestion
"This product's company cannot be changed as long as there are quantities of it  "
```

Reply:
Sorry, I didn't see the previous comment about that, I fixed now, thanks :)